contracts/StakeManager.sol

```diff
@@ -50,8 +53,15 @@ contract StakeManager is Ownable {
 
     uint256 public currentEpoch;
     uint256 public pendingReward;
+
+    uint256 public pendingMPToBeMinted;
     uint256 public totalSupplyMP;
     uint256 public totalSupplyBalance;
+    uint256 public totalMPPerEpoch;
```

Comment:
We should rename this, as this is not actually "total MP per epoch", but "total possible MP for all epochs"

Reply:
Sort of... Because this value is increased when stake, but decreased when finalize epoch.